package com.company;

 /*public class exxception1 {
    public static void main(String[] args){
        try{
            int b=1/0;

        }
        catch(ArithmeticException e){
            System.out.println(e);
            System.out.println("error");
        }
        System.out.println("executed");
    }
} */

/* public class exxception1{
    public static void main(String[] args){
        try{
            int b=25/5;
            System.out.println(b);
        }
        catch (ArithmeticException e){
            System.out.println(e);
            System.out.println("error");
        }
        System.out.println("executed");
    }
} */

